tools/BUILD

```diff
@@ -23,6 +23,17 @@ buildifier(
     mode = "fix",
 )
 
+#
+# When running the dev_cluster the default data path is the working directory
+# where the python script is run. Bazel runs the commands in a runfiles
+# directory which makes it inconvenient to interact with the cluster data
+# directory.
+#
+# When running you can pass in `-d` so that the data directory will be your
+# current working directory.
+#
+#     bazel run //tools:dev_cluster -- --nodes 1 -d ${PWD}/data
```

Comment:
We can fix the default in the dev tool if you want.

`bazel run` populates `BUILD_WORKSPACE_DIRECTORY` which will be the root of repo. This is usually done for tooling that modifies the monorepo, but could work here too.

Reply:
oh sweet. yeh ok do this